CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to dxmentorship.com Rebuild
+
+Thank you for your interest in contributing to the dxmentorship.com rebuild project. We welcome your contributions to help make the project more efficient, user-friendly, and informative for mentees.
+
+## Types of Contributions
+
+You can contribute to this project in several ways, including but not limited to:
+
+- Reporting issues or bugs
+- Enhancing the website's functionality
+- Improving the user interface and user experience (UI/UX)
+- Adding content, such as details of previous graduates or curriculum information
+- Reviewing and improving code
+- Writing documentation
+
+## Getting Started
+
+Before you begin contributing, please follow these steps:
+
+1. Fork the repository to your GitHub account.
+2. Clone your forked repository to your local machine and add upstream remote:
+
+```bash
+git clone https://github.com/dxmentorship/dxmentorship.git
+cd dxmentorship
+git remote add upstream https://github.com/dxmentorship/dxmentorship.git
+```
+
+
+## How to Contribute
+
+1. **Reporting Issues**: If you encounter a bug, issue, or have a feature request, please create an issue on our GitHub repository. When creating an issue, provide as much detail as possible, including steps to reproduce.
+
+2. **Pull Requests**: If you want to contribute code or enhancements, please follow these guidelines:
+   - Create a new branch for your feature or fix.
+   - Make your changes and commit them.
+   - Test your changes and ensure that they do not introduce new issues.
+   - Submit a pull request to the main repository with a clear description of your changes.
+
+3. **Code Review**: We welcome code reviews from contributors. If you are not a developer but would like to help review code or documentation, please feel free to comment on pull requests.
+
+4. **Documentation**: If you are improving documentation, please follow the existing style and make sure your documentation is clear and concise.
+
+## Code of Conduct
+
+By participating in this project, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Contact
+
+If you have questions, concerns, or need to contact the project team, you can reach out to [maintainer email or contact information].
```

Comment:
We have a discord community. I think that would be a preferable point of contact. 

https://discord.com/invite/dUQyb2mk4s